components/Modal.tsx

```diff
@@ -18,24 +19,21 @@ const Modal = ({ name, itemId, message }: ModalProps) => {
 
   return (
     <>
-      {!modal && (
-        <button className='button button-rounded my-2' onClick={toggleModal}>
-          {name}
-        </button>
-      )}
+      <button className='button button-rounded my-2' onClick={toggleModal}>
+        {name}
+      </button>
       {modal && (
```

Comment:
What's this state monitoring? I know it's legacy but can you please change it to be more descriptive?

It looks to me at first read like this might be controlling whether the modal is open so maybe we can call it `isOpen`? You get your pick but please make this fit our conventions 😵‍💫